HotelApp/Models/Hotel.cs

```diff
@@ -16,6 +16,7 @@ public class Hotel
         public string City { get; set; }
```

Comment:
Sve ove modele koji su ti za bazu bi trebao stavit u neki drugi folder. Ja bi to stavio u DbContext/Entities. 

Ovaj Models folder će ti biti za Dto (Data transfer object), iliti request i response modele. Na većim projektima se to čak razdvaja u posebne projekte da bi se moglo lakše testirat. Možeš pogledat neke edukacije za design patterne u slobodno vrijeme, to će ti bit dosta kasnije, za ovo tu nije previše bitno. 

Ali sigurno bi te modele onda prebacio u DbContext/Entities folder da ti sav taj dio za bazu bude na jednom mjestu